templates/glauth.cfg.j2

```diff
@@ -0,0 +1,30 @@
+debug = true
+
+[ldap]
+  enabled = true
+  listen = "0.0.0.0:{{ ldap_port }}"
+
+[ldaps]
+  enabled = false
+
+[backend]
+  datastore = "plugin"
+  plugin = "{{ postgres_plugin }}"
+  pluginhandler = "NewPostgresHandler"
+  baseDN = "{{ baseDN }}"
+  database = "postgres://{{ db_info.get('username') }}:{{ db_info.get('password') }}@{{ db_info.get('endpoints') }}/{{ db_info.get('database_name') }}?sslmode=disable"
+
+[behaviors]
+  IgnoreCapabilities = {{ ignore_capabilities  }}
+  LimitFailedBinds = {{ limited_failed_binds }}
+  NumberOfFailedBinds = {{ number_of_failed_binds }}
+  PeriodOfFailedBinds = {{  period_of_failed_binds }}
+  BlockFailedBindsFor = {{ block_failed_binds_for }}
+  PruneSourceTableEvery = {{ prune_source_tables_every }}
+  PruneSourcesOlderThan = {{ prune_sources_older_than }}
```

Comment:
Are we sure we want to expose these as config? Isn't it our job as charmers to configure these to make the user's life easier? (cc @massigori)

Reply:
On one hand yes, however: there are default values for all these, so users don't have to interact with these configs if they don't want to. If we don't make these configs exportable, then it's really hard to change them for the user.